### ISSUE TITLE
Skyconnect Failed to probe

### ISSUE THREAD

**Issue by Toestesen** [home-assistant/addons#3171]
### Describe the issue you are experiencing

I'm running homeassisstant on RPI 4b 4 gb and every seems to work fine. 
The only problem is I can't get my skyconnect to connect with ZHA
The skyconnect is connected with af 1,8 m extentioncable. 

I've tried to flash the skyconncent sereveral times with the newst update(EZSP) 
I've tried to reflash my SSD with homeassistant and start from scratch, so no add-on could block the integration.

i've tried both with and without Silicon Labs Multiprotocol tried to set the configuration baudrate to 115200

Still I can't figure out why it wont work. 

Can anyone from my Silicon Labs Multiprotocol log see what i'm doing wrong? 

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

Silicon Labs Multiprotocol

### What is the version of the add-on?

version: 2.3.1

### Steps to reproduce the issue

1.Insert skyconnect with af extensioncable of 1,8
2. Go to hadware og configure skyconnect  - setup Silicon Labs Multiprotocol
3. Ad integration - ZHA - Serial Device Path - chose my skyconnect: /dev/ttyUSB0 - SkyConnect v1.0, s/n: 6c3f2d997b91ed119ce5cad13b20a988 - Nabu Casa
4. choose radio type - EZSP
5. Serial divice path: /dev/ttyUSB0 
6.baudrate 115200
7. data flow control: software
8. And then: FAILED TO CONNECT
...

### System Health information

[12:53:09] INFO: The otbr-web is disabled.
s6-rc: info: service mdns: starting
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service mdns successfully started
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/check-cpcd-shm.sh
[12:53:09] INFO: Starting mDNS Responder...
Default: mDNSResponder (Engineering Build) (Aug  4 2023 21:48:56) starting
Default: mDNS_AddDNSServer: Lock not held! mDNS_busy (0) mDNS_reentrancy (0)
cont-init: info: /etc/cont-init.d/check-cpcd-shm.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service banner: starting
-----------------------------------------------------------
 Add-on: Silicon Labs Multiprotocol
 Zigbee and OpenThread multiprotocol add-on
-----------------------------------------------------------
 Add-on version: 2.3.1
 You are running the latest version of this add-on.
 System: Home Assistant OS 10.4  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.8.1
 Home Assistant Supervisor: 2023.08.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
s6-rc: info: service banner successfully started
s6-rc: info: service universal-silabs-flasher: starting
[12:53:11] INFO: Checking /dev/ttyUSB0 identifying SkyConnect v1.0 from Nabu Casa.
[12:53:11] INFO: Starting universal-silabs-flasher with /dev/ttyUSB0
2023-08-09 12:53:12 homeassistant universal_silabs_flasher.flash[181] INFO Extracted GBL metadata: NabuCasaMetadata(metadata_version=1, sdk_version='4.3.1', ezsp_version=None, ot_rcp_version=None, fw_type=<FirmwareImageType.RCP_UART_802154: 'rcp-uart-802154'>, baudrate=460800)
2023-08-09 12:53:12 homeassistant universal_silabs_flasher.flasher[181] INFO Probing ApplicationType.GECKO_BOOTLOADER at 115200 baud
2023-08-09 12:53:14 homeassistant universal_silabs_flasher.flasher[181] INFO Probing ApplicationType.CPC at 460800 baud
2023-08-09 12:53:14 homeassistant universal_silabs_flasher.cpc[181] WARNING Failed to parse buffer bytearray(b'\xf8'): ValueError('Invalid flag')
2023-08-09 12:53:18 homeassistant universal_silabs_flasher.flasher[181] INFO Probing ApplicationType.CPC at 115200 baud
2023-08-09 12:53:23 homeassistant universal_silabs_flasher.flasher[181] INFO Probing ApplicationType.CPC at 230400 baud
2023-08-09 12:53:27 homeassistant universal_silabs_flasher.flasher[181] INFO Probing ApplicationType.EZSP at 115200 baud
2023-08-09 12:53:28 homeassistant bellows.uart[181] ERROR CRC error in frame b'c102030a527e' (b'0a52' != b'8b5a')
2023-08-09 12:53:32 homeassistant universal_silabs_flasher.flasher[181] INFO Probing ApplicationType.SPINEL at 460800 baud
Error: Failed to probe running application type
s6-rc: warning: unable to start service universal-silabs-flasher: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service mdns: stopping
s6-rc: info: service banner: stopping
Default: mDNSResponder (Engineering Build) (Aug  4 2023 21:48:56) stopping
s6-rc: info: service banner successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
[12:53:37] INFO: mDNS ended with exit code 4 (signal 0)...
s6-rc: info: service mdns successfully stopped 

### Anything in the Supervisor logs that might be useful for us?

```txt
Home Assistant Core
2023-08-09 12:21:07.951 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Home Assistant SkyConnect' for homeassistant_sky_connect integration not ready yet: Failed to start the Silicon Labs Multiprotocol add-on: Unknown error, see supervisor; Retrying in background
2023-08-09 12:24:19.224 WARNING (MainThread) [zigpy_deconz.api] No response to 'Command.read_parameter' command with seq id '0x02'
2023-08-09 12:24:43.577 ERROR (MainThread) [bellows.uart] CRC error in frame b'c102030a527e' (b'0a52' != b'8b5a')
2023-08-09 12:24:46.582 ERROR (MainThread) [bellows.uart] CRC error in frame b'c102030a527e' (b'0a52' != b'8b5a')
2023-08-09 12:24:55.977 ERROR (MainThread) [bellows.uart] CRC error in frame b'c102030a527e' (b'0a52' != b'8b5a')
```

**Comment by darkxst** [comment]
Multiprotocol is failing to flash the Multipan firmware which is required for it too work.
> I've tried to flash the skyconncent sereveral times with the newst update(EZSP)

How did you try to flash EZSP updates? did this fail also?


> 2023-08-09 12:53:28 homeassistant bellows.uart[181] ERROR CRC error in frame b'c102030a527e' (b'0a52' != b'8b5a')

This error may indicate a hardware issue with USB connection, try another USB port without the USB cable and see if that helps.

Alternatively try on another computer and use the webflasher to install Multipan firmware:
https://skyconnect.home-assistant.io/firmware-update/



> 6.baudrate 115200
> 7. data flow control: software

These settings are wrong (but likely not causing your issue), if you get past flashing the device successfully you will want
baud 460800 + hardware flow control

**Comment by Toestesen** [comment]
Thanks darkxst

Its is exactly the web flasher i've used to re-flash/update the skyconnect stick. When I plug it into my chromebook the process goes right through without any problems. 

I've mainly used the EZSP update, but it dosn't help to try the multipan(RCP) firmware. 
Only Silicon Labs Multiprotocol was not able to start up.

I've tried all the different usb port and with and without cable and also different extension cables.

It seems like they are working. I can boot HA for my USB connected SSD and its also possible to connect zwave.me stick and make it work. 
I was thinking if the dogle was not getting enough power, but changing to a more powerfull powersuply wasn't changing anything. 

I've borrowed af sonoff dogle E and get a similar error when trying to set it op. 
It is reconized when trying set it up in ZHA, but again it fails to connect in the end. 

![Screenshot 2023-08-10 08 05 45](https://github.com/home-assistant/addons/assets/140346815/d9fbf946-0f08-4327-ac7c-2bbd05407cb9)

The log goes like:
Logger: bellows.uart
Source: runner.py:179
First occurred: 7:51:26 AM (2 occurrences)
Last logged: 7:51:58 AM

CRC error in frame b'c102030a527e' (b'0a52' != b'8b5a')


So its seems like my problem relates not only skyconnect but all zigbee devices?
